testing/ostest/mutex.c

```diff
@@ -57,8 +57,15 @@
 
 static pthread_mutex_t mut;
 static volatile int my_mutex = 0;
-static unsigned long nloops[2] = {0, 0};
-static unsigned long nerrors[2] = {0, 0};
+static unsigned long nloops[2] =
+  { 0,
```

Comment:
```suggestion
  {
    0,
```
nit: similar to the below format

Reply:
Thanks, fixed.